src/main/java/bio/terra/common/CloudPlatformWrapper.java

```diff
@@ -63,6 +64,8 @@ public <T> T choose(Map<CloudPlatform, Supplier<T>> cloudMap) {
 
   public abstract <T> T choose(Supplier<T> gcp, Supplier<T> azure);
 
+  public abstract <T> T choose(T gcp, T azure);
```

Comment:
We now have many cases where `choose()` is being used to select between values, so I added another overload to avoid having to create a function that just returns a value.